extensions/wikia/Search/classes/Services/CombinedSearchService.php

```diff
@@ -213,182 +266,215 @@ public function queryPhraseForWikias( $query, $hubs, $lang ) {
 				break;
 			}
 		}
 		$wikias = array_slice( $wikias, 0, self::CROSS_WIKI_RESULTS );
 		$timer->stop();
+
 		return $wikias;
 	}
 
 	public function phraseSearchForWikias( $query, $langs, $hubs ) {
 		$timer = Time::start( [ __CLASS__, __METHOD__ ] );
-		$wikias = [ ];
+		$wikias = [];
 		foreach ( $langs as $lang ) {
 			$crossWikiResults = $this->queryPhraseForWikias( $query, $hubs, $lang );
 			foreach ( $crossWikiResults as $wikiSearchResult ) {
 				//PLA-1116: set to false
-				$wikias[ ] = $this->processWiki( $wikiSearchResult, false );
+				$wikias[] = $this->processWiki( $wikiSearchResult, false );
 			}
 			if ( sizeof( $wikias ) >= self::CROSS_WIKI_RESULTS ) {
 				break;
 			}
 		}
 		$timer->stop();
+
 		return $wikias;
 	}
 
 	/**
 	 * @param $query
 	 * @param $hubs
 	 * @param $lang
+	 *
 	 * @return array
 	 */
 	public function queryForWikias( $query, $hubs, $lang ) {
 		$crossWikiSearchConfig = new Config;
-		$crossWikiSearchConfig->setQuery( $query )
-			->setLimit( self::CROSS_WIKI_RESULTS )
-			->setPage( 1 )
-			->setRank( 'default' )
-			->setInterWiki( true )
-			->setCommercialUse( $this->getHideNonCommercialContent() )
-			->setLanguageCode( $lang );
+		$crossWikiSearchConfig->setQuery( $query )->setLimit( self::CROSS_WIKI_RESULTS )->setPage( 1 )->setRank(
+				'default'
+			)->setInterWiki( true )->setCommercialUse( $this->getHideNonCommercialContent() )->setLanguageCode( $lang );
 		if ( !empty( $hubs ) ) {
 			$crossWikiSearchConfig->setHubs( $hubs );
 		}
 		$crossWikiResultSet = ( new Factory )->getFromConfig( $crossWikiSearchConfig )->search();
-		$crossWikiResults = $crossWikiResultSet->toArray( [ "sitename_txt", "url", "id", "description_txt", "lang_s", "score", "description_txt" ] );
+		$crossWikiResults = $crossWikiResultSet->toArray(
+			[ "sitename_txt", "url", "id", "description_txt", "lang_s", "score", "description_txt" ]
+		);
+
 		return $crossWikiResults;
 	}
 
 	protected function processWiki( $wikiInfo, $addTopArticles = false ) {
-		$outputModel = [ ];
-		$outputModel[ 'wikiId' ] = (int)$wikiInfo[ 'id' ];
-		$outputModel[ 'name' ] = $wikiInfo[ 'sitename_txt' ][ 0 ]; // this is multivalue field
-		$outputModel[ 'url' ] = $wikiInfo[ 'url' ];
-		$outputModel[ 'lang' ] = $wikiInfo[ 'lang_s' ];
-		$outputModel[ 'snippet' ] = $wikiInfo[ 'description_txt' ];
-		$outputModel[ 'wordmark' ] = $this->wikiService->getWikiWordmark( $outputModel[ 'wikiId' ] );
+		$outputModel = [];
+		$outputModel['wikiId'] = (int) $wikiInfo['id'];
+		$outputModel['name'] = $wikiInfo['sitename_txt'][0]; // this is multivalue field
+		$outputModel['url'] = $wikiInfo['url'];
+		$outputModel['lang'] = $wikiInfo['lang_s'];
+		$outputModel['snippet'] = $wikiInfo['description_txt'];
+		$outputModel['wordmark'] = $this->wikiService->getWikiWordmark( $outputModel['wikiId'] );
 //		$images = $this->wikiService->getWikiImages( $outputModel['wikiId'], self::IMAGE_SIZE, self::IMAGE_SIZE );
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this commented out code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS125)